### DISCUSSION TITLE
settle on the acronyms now

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1859] [Ideas]
I'd like to see a nomenclature for referencing the flavor of Vanjaro when making a post because it is useful to know the context of a topic, especially as the number of discussion items grow into the thousands. Something like,

VN: Vanjaro Native
VB: Vanjaro Business
VE: Vanjaro Enterprise

**Comment by meetmandeep** [comment]
We can stick with the names that are shown in the About Us menu. So use:

VP: Vanjaro Platform (Open Source)
VB: Vanjaro Business
VE: Vanjaro Enterprise

Keep in mind that we will have a dedicated support/forums for Business & Enterprise customer. Ultimately, all discussions on GitHub should be relevant to Vanjaro Platform only.
